### PR TITLE
Setuptools installation

### LINKED ISSUE #179: Allow installing using setuptools
Allow installing using setuptools

### PR DESCRIPTION
**Pull request by t0xic0der**
Fixes #179

### CODE REVIEW

**Comment by davidkirwan** [comment]
Looks good, I ran through the README steps, seems to all work as expected!

**Review by davidkirwan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
-6. `python3 main.py --help`
-7. `python3 main.py -p 9696 -4`
+Python Flask project used to aggregate and distribute IRC meeting minutes and logs for the Fedora Project
 
-## How to test changes locally using Podman?
+## How to set up a local development environment?
 
-1. Build the image `podman build . -f Dockerfile -t fedora-easyfix:latest`.
-2. Start the server `podman run -p 9696:9696 -ti fedora-easyfix:latest -p 9696 -4`
+1. Download https://drive.google.com/file/d/1iOKEgyHMtpnCwsuZa5ufT8gI9-ntuOVc/view?usp=sharing.
```
Comment (Zlopez) [inline README.md:7]: You can't access this link without Google account

**Review by Zlopez — COMMENTED** [review]
Just one comment for the link mentioned in README.md. Otherwise looks good.

File: README.md (hunk 1 above)
Comment (t0xic0der) [reply README.md]: I have now moved the archive to Mega. Please let me know if any more changes are needed.

**Review by Zlopez — APPROVED** [review]

**Pull request merged**